### DISCUSSION TITLE
Vwma CandlePart parameter

### DISCUSSION THREAD

**Discussion by bobbyhood99** [DaveSkender/Stock.Indicators#1236] [Ideas]
The Vmwa calculation is hard coded to always use the Close property of the candle.  Is it possible to extend that so other CandlePart options can be used?

**Comment by DaveSkender** [comment]
I think the reason I did not implement this one like most other moving averages, to allow selecting any `CandlePart` with a `Use(candlePart)` option, is because it relies on both (close) price and volume for weights.  It may not make sense if you were to select `Low` or `Volume` parts, for example.

I’ve added it to the backlog to evaluate:

- #1237
